askama_derive/src/parser.rs

```diff
@@ -668,7 +670,29 @@ macro_rules! expr_prec_layer {
         }
     }
 }
-
+#[cfg(feature = "localization")]
```

Comment:
I actually would remove this feature gate in here and instead add a feature gated [`fail()`](https://docs.rs/nom/7.1.1/nom/combinator/fn.fail.html) after `ws(tag("("))`, so the name `localize` is reserved with and without the feature.

Reply:
I added it to expr_any but it has the same effect